src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="./main.js">
+    </script>
+    <form class="block">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-filed">
+            Surname:
+          <input type="text" name="surname" autocomplete="of">
+        </label>
+
+        <label class="form-filed">
+            Name:
+          <input type="text" name="name"  autocomplete="off">
+        </label>
+
+        <label class="form-filed">
+            How old are You?
+          <input
+            autocomplete="off"
+            type="number" 
+            name="age" 
+            value="12"
+            min="1" 
+           max="100"
+          >
+        </label>
+
+        <label class="form-filed">
+            Full date of birth:
+          <input type="date" name="date-of-birth"  autocomplete="off">
+        </label>
+
+        <label class="form-filed">
+            I accept the term of the agreement
+          <input type="checkbox" name="checkbox">
+        </label>
+
+      </fieldset>
+    </form> 
+
+    <form class="block">
+      <fieldset>
+
+        <legend>Registration</legend>
+
+        <label class="form-filed">
+            E-mail:
+          <input 
+            autocomplete="off"
+            type="email" 
+            placeholder="email@example.com" 
+            required
+          >
+        </label>
+
+        <label class="form-filed">
+            Password:
+          <input 
+            autocomplete="off"
+            type="password" 
+            minlength="8" 
+            maxlength="20"
+            required
+          >
+        </label>
+
+      </fieldset>
+    </form>
+
+    <form class="block">
+      <fieldset>
+
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-filed">
+            Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="LoveCats?" 
+              value="Y"
+            >
+              yes
+          </label>
+
+          <label>
+            <input 
+                type="radio" 
+                name="LoveCats?"
+                value="N"
+            >
```

Comment:
```suggestion
            <input 
              type="radio" 
              name="LoveCats?"
              value="N"
            >
```
Везде должно быть одинаковое количество отступов(2)